setup.py

```diff
@@ -25,6 +25,7 @@
 
 # Build the setup
 setup(
+    python_requires=">=3.7",
```

Comment:
@sapetnioc Is 3.7 the desired value for the minimal Python version to support?

Reply:
I am not sure it is high enough. If I remember well, the syntax `list[...]` was introduced in 3.9. This syntax is used in populse_db.

Reply:
All tests are run using 3.9. Then I will bump the Python version to 3.9.